### ISSUE TITLE
New project create fails with IllegalStateException

### ISSUE THREAD

**Issue by uncleLem** [ArcBees/gwtp-idea-plugin#61]
After selecting archetype (or just hitting 'Next' without any archetype selected) I'm getting this:

```
Cannot parse object because no supported Content-Type was not specified in response. Content-Type was 'text/html'.
java.lang.IllegalStateException: Cannot parse object because no supported Content-Type was not specified in response. Content-Type was 'text/html'.
    at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
    at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
    at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance(Constructor.java:408)
    at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:77)
    at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:102)
    at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:190)
    at com.jayway.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:78)
    at com.jayway.restassured.internal.mapping.ObjectMapping$deserialize.call(Unknown Source)
    at com.jayway.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:206)
    at com.jayway.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:153)
    at com.arcbees.plugin.idea.wizards.createproject.FetchArchetypes.fetchArchetypes(FetchArchetypes.java:51)
    at com.arcbees.plugin.idea.wizards.createproject.CreateProjectWizard$1.run(CreateProjectWizard.java:113)
    at com.intellij.openapi.application.impl.ApplicationImpl$8.run(ApplicationImpl.java:406)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:745)
    at org.jetbrains.ide.PooledThreadExecutor$1$1.run(PooledThreadExecutor.java:56)
```

**Comment by MichealZub** [comment]
Reproduced in IntelliJ IDEA 14.1.3 Build #IU-141.1010.3
Plugin version: 1.2.0-beta4

**Comment by olafleur** [comment]
There is a fix in the 1.2.0 branch (which is the main branch). We should release officially a new version of the plugin soon, but you can test in the meantime by downloading it here on GitHub.

**Comment by olafleur** [comment]
Fixed in 1.2.0. Closing.
